src/quo/components/list_items/token_network/component_spec.cljs

```diff
@@ -0,0 +1,74 @@
+(ns quo.components.list-items.token-network.component-spec
+  (:require [quo.components.list-items.token-network.view :as token-network]
+            [quo.foundations.colors :as colors]
+            [quo.foundations.resources :as resources]
+            [test-helpers.component :as h]))
+
+(h/describe "List items: Token network"
+  (h/test "default state"
+    (h/render [token-network/view
```

Comment:
I'd suggest adding a test to check what happens when no props passed to the component, like `[token-network/view  {}]`. Quite often that helps to find crashes. Those crashes probably won't happen in production but are very irritating when you develop something using the component and instantiate it without props first.